buttons/button.h

```diff
@@ -3,19 +3,56 @@
 
 #include "button_base.h"
 
+// For all setups following the wiring diagram (V2) described on https://fredrik.hubbe.net/lightsaber/v5/
+// 
+// momentary buttons are expected to be connected to gnd and 1 of the Proffieboard/Teensy input pins
+//
+// Usage example, put this in the button config section: 
+// Button PowerButton(BUTTON_POWER, powerButtonPin, "pow");  
+//
+
 class Button : public ButtonBase {
 public:
+
   Button(enum BUTTON button, int pin, const char* name) : ButtonBase(name, button), pin_(pin) {
     pinMode(pin, INPUT_PULLUP);
 #ifdef ENABLE_SNOOZE
     snooze_digital.pinMode(pin, INPUT_PULLUP, RISING);
 #endif
   }
+
 protected:
+  uint8_t pin_;
+
   bool Read() override {
     return digitalRead(pin_) == LOW;
   }
-  uint8_t pin_;
 };
 
+
+// For special setups were you want to hook up a momentary button to VCC(3.3V) or Vbat
+// and the other end to the inputs of the Proffieboard / Teensy 
+// 
+// For Proffieboards V2 all inputs can handle Vbat, except for Data3 / blade3pin
```

Comment:
This is incorrect, Data4 is the one that not 5V tolerant.

Reply:
As stated in the V2 config file;
(or did you reroute the pins different?)

blade3Pin = 17, // PA4 -> not 5V tolerant TT_a
blade4Pin = 0, // PB3 FT_la

From the DS11449 Rev 5 STM32l433cc data sheet;

14 14 G6 E5 20 20 H3 29 M3 **_PA4_** I/O **_TT_a_** -SPI1_NSS,SPI3_NSS,USART2_CK,SAI1_FS_B,LPTIM2_OUT,EVENTOUT 
COMP1_INM,
COMP2_INM,
ADC1_IN9,
DAC1_OUT1

39 39 A3 A5 55 54 A5 89 A8 **_PB3_** (JTDOTRACESWO) I/O **_FT_la_** (3) JTDO-TRACESWO,TIM2_CH2,SPI1_SCK,SPI3_SCK, USART1_RTS_DE,
LCD_SEG7,
SAI1_SCK_B,
EVENTOUT
COMP2_INM

Reply:
The comments are not accurate it seems.  (I will fix that)
blade4Pin is PA4 and blade3Pin is PB3

Reply:
Ah, That makes sense. Also the fact that Data4 is the only DAC capable output and PA4 being only 1 of 2 pins that can do this...

Modified the comment to Data4 not being 5V tolerant.